none1.html

```diff
@@ -0,0 +1,73 @@
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="bootstrap.min.css"> </link>
+    <style type="text/css" media='screen,projection'>
+
+    @import = url("https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css");
```

Comment:
Instead of:
```
@import = url("https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css")
``` 
use:
```
<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css" />
```
and put it _outside_ `<style></style>` tag, not inside. @import also works, but is not good because it can only download one file at a time. Also, it's `@import url(...`, there's no `=` after `@import.`

Make that change everywhere else too.